google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsConstants.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import com.google.api.core.InternalApi;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.sdk.metrics.Aggregation;
+import io.opentelemetry.sdk.metrics.InstrumentSelector;
+import io.opentelemetry.sdk.metrics.InstrumentType;
+import io.opentelemetry.sdk.metrics.View;
+import java.util.Map;
+import java.util.Set;
+
+/** Defining Bigtable builit-in metrics scope, attributes, metric names and views. */
+@InternalApi
+public class BuiltinMetricsConstants {
+
+  public static final AttributeKey<String> PROJECT_ID = AttributeKey.stringKey("project_id");
+  public static final AttributeKey<String> INSTANCE_ID = AttributeKey.stringKey("instance");
+  public static final AttributeKey<String> TABLE_ID = AttributeKey.stringKey("table");
+  public static final AttributeKey<String> CLUSTER_ID = AttributeKey.stringKey("cluster");
+  public static final AttributeKey<String> ZONE_ID = AttributeKey.stringKey("zone");
+  static final AttributeKey<String> CLIENT_UID = AttributeKey.stringKey("client_uid");
+
+  public static final AttributeKey<String> APP_PROFILE = AttributeKey.stringKey("app_profile");
+  public static final AttributeKey<Boolean> STREAMING = AttributeKey.booleanKey("streaming");
```

Comment:
why are some public and others are pkg private?

Reply:
We need to access APP_PROFILE in EnhancedBigtableStub and STREAMING is used in BuiltinMetricsIT. Both are under a different package.

Reply:
Add a comment.